Source/SVWidgetsLib/QtSupport/QtSDocServer.cpp

```diff
@@ -47,10 +47,10 @@
 QtSDocServer* QtSDocServer::self = nullptr;
 
 static HttpListener* httpListener = nullptr;
-static QSettings* sessionSettings = nullptr;
 static HttpSessionStore* sessionStore = nullptr;
-static QSettings* fileSettings = nullptr;
-static QSettings* listenerSettings = nullptr;
+
+static ServerSettings* s_ServerSettings = nullptr;
```

Comment:
Raw global pointers should be avoided unless there is a very good reason to use them.  Also, `m_ServerSettings` was already declared in the header.  `s_ServerSettings` is redundant and lacks any form of memory management (no delete statement and not a std::shared_ptr).

Reply:
Done and fixed